Maestro/subscriptions.json

```diff
@@ -646,8 +646,15 @@
         "https://github.com/dotnet/arcade/blob/release/**/*",
         "https://github.com/dotnet/diagnostics/blob/master/**/*",
         "https://github.com/dotnet/diagnostics/blob/release/**/*",
+        "https://github.com/dotnet/core-sdk/blob/master/**/*",
+        "https://github.com/dotnet/core-sdk/blob/release/**/*",
+        "https://github.com/dotnet/cli/blob/master/**/*",
```

Comment:
I was under the impression we had CLI already.

Reply:
Only in the devdiv account.  This is for the dotnet account.